internal/acceptance/cli/cli.go

```diff
@@ -147,6 +147,9 @@ func ecCommandIsRunWith(ctx context.Context, parameters string) (context.Context
 		logger.Log(err)
 	}
 
+	logger.Logf("Stdout: %v", stdout.String())
+	logger.Logf("Stderr: %v", stderr.String())
+
```

Comment:
There is already `logOutput` that could be used instead, it prints much more. Having this here the stdout/stderr will be printed twice on unexpected exit status, might be a bit confusing

Reply:
Looks like this has been dealt with via ed0db955c00d0c51c0f1eb4ec96b584ff0aa2fec .